### PR TITLE
Adding basic sources configuration (ping source)

### PR DESCRIPTION
**Pull request by salaboy**
Fixes: https://github.com/knative/eventing/issues/5529 

This issue is related to the 1.0 GA release, please review as soon as possible. 

## Proposed Changes 

- Adds section for Sources Configurations and describe the PingSource configMap as it is the only source that can be configured (as far as I know) 
- This PR can serve as the basics for other configurations to be added later.

### CODE REVIEW

File: docs/admin/eventing/sources-configuration.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Sources Configurations
+
+Each Knative Source can be configured depending on how it is generating events. This document contains details about how to configure the available Sources that can be configured.
+
+## Ping Source Configurations
```
Comment (lionelvillard) [inline docs/admin/eventing/sources-configuration.md:5]: Ping Source -> PingSource

File: docs/admin/eventing/sources-configuration.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Sources Configurations
```
Comment (snneji) [inline docs/admin/eventing/sources-configuration.md:1]:
```suggestion
# Configure event source defaults
```

File: config/nav.yml [hunk 3]
```diff
@@ -60,6 +60,7 @@ nav:
         - Knative Eventing configuration:
           - Configure Broker defaults: admin/eventing/broker-configuration.md
           - Configure Kafka Channel defaults: admin/eventing/kafka-channel-configuration.md
+          - Configure Sources defaults: admin/eventing/sources-configuration.md
```
Comment (snneji) [inline config/nav.yml:63]:
```suggestion
          - Configure event source defaults: admin/eventing/sources-configuration.md
```

File: docs/admin/eventing/sources-configuration.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Each Knative Source can be configured depending on how it is generating events. This document contains details about how to configure the available Sources that can be configured.
+
+## Ping Source Configurations
+
+A [Ping Source](../../developer/eventing/sources/ping-source/README.md) is an event source that produces events with a fixed payload on a specified cron schedule. You can find how to create new Ping Source and the available parameters [here](../../developer/eventing/sources/ping-source/reference.md).
+
+Besides the parameters that can be configured in the `PingSource` resources, there is a global `ConfigMap` called `config-ping-defaults`.
+
+This `ConfigMap` allows you to change the maximun amount of data that the `PingSource` will add to the produced CloudEvents.
```
Comment (snneji) [inline docs/admin/eventing/sources-configuration.md:11]:
```suggestion
In addition to the parameters that you can configure in the PingSource resource, there is a global ConfigMap called `config-ping-defaults`.
This ConfigMap allows you to change the maximum amount of data that the PingSource adds to the CloudEvents it produces.
```

File: docs/admin/eventing/sources-configuration.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+The `dataMaxSize` parameter allows you to set the max number of bytes allowed to be sent for message excluding any base64 decoding. The default value (-1) is no limit set for data.
```
Comment (snneji) [inline docs/admin/eventing/sources-configuration.md:13]:
```suggestion
The `dataMaxSize` parameter allows you to set the maximum number of bytes allowed to be sent for a message excluding any base64 decoding. The default value, `-1`, sets no limit for data.
```

File: docs/admin/eventing/sources-configuration.md (hunk 1 above)
Comment (snneji) [inline docs/admin/eventing/sources-configuration.md:5]:
```suggestion
## Configure defaults for PingSource
```

File: docs/admin/eventing/sources-configuration.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-ping-defaults
+  namespace: knative-eventing
+data:
+  dataMaxSize: -1
+```
+
+You can edit this `ConfigMap` by running:
```
Comment (snneji) [inline docs/admin/eventing/sources-configuration.md:25]:
```suggestion
You can edit this ConfigMap by running the command:
```

File: docs/admin/eventing/sources-configuration.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
+# Sources Configurations
+
+Each Knative Source can be configured depending on how it is generating events. This document contains details about how to configure the available Sources that can be configured.
```
Comment (snneji) [inline docs/admin/eventing/sources-configuration.md:3]:
```suggestion
This topic describes how to configure defaults for Knative event sources. You can configure event sources depending on how they generate events.
```

File: docs/admin/eventing/sources-configuration.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+## Ping Source Configurations
+
+A [Ping Source](../../developer/eventing/sources/ping-source/README.md) is an event source that produces events with a fixed payload on a specified cron schedule. You can find how to create new Ping Source and the available parameters [here](../../developer/eventing/sources/ping-source/reference.md).
```
Comment (snneji) [inline docs/admin/eventing/sources-configuration.md:7]:
```suggestion
PingSource is an event source that produces events with a fixed payload on a specified cron schedule. For how to create a new PingSource, see [Creating a PingSource object](../../developer/eventing/sources/ping-source/README.md).
For the available parameters, see [PingSource reference](../../developer/eventing/sources/ping-source/reference.md).
```

**Review by snneji — CHANGES_REQUESTED** [review]
A few style changes

**Comment by salaboy** [comment]
@lionelvillard @snneji I think all suggestions were applied now

**Comment by lionelvillard** [comment]
/approve
LGTM

@snneji do you want to take another look?

**Comment by snneji** [comment]
> @abrennan89 @snneji while reviewing this PR I've noticed inconsistencies in the way sources are named in the developer guide navbar:
> 
> Screen Shot 2021-10-06 at 9 09 09 AM
> .

@lionelvillard The titles that are in camel case are for topics that have recently been edited. Personally, I prefer referring to these sources uses their resource name in camel case, and would like the remaining topics to be updated. Not sure if others agree though. 

If you like you can make an issue to make these consistent.

**Pull request merged**